tests/blackbox/stratisd_cert.py

```diff
@@ -70,6 +99,162 @@ def test_get_managed_objects(self):
+        """
+        self.assertEqual(StratisDbus.fs_list(), {})
+
+    def test_pool_create(self):
+        """
+        Test creating a pool.
+        """
+        pool_name = p_n()
+        self.assertEqual(StratisDbus.pool_create(pool_name, DISKS)[1], 0)
+
+    def test_pool_add_cache(self):
+        """
+        Test adding cache to a pool.
+        """
+        pool_name = p_n()
+        assert StratisDbus.pool_create(pool_name, DISKS)
+        self.assertEqual(StratisDbus.pool_add_cache(pool_name, DISKS)[1], 0)
+
+    def test_pool_add_data(self):
+        """
+        Test adding data to a pool.
+        """
+        pool_name = p_n()
+        assert StratisDbus.pool_create(pool_name, DISKS)
+        self.assertEqual(StratisDbus.pool_add_data(pool_name, DISKS)[1], 0)
+
+    def test_pool_list_not_empty(self):
+        """
+        Test listing an non-existent pool.
+        """
+        make_test_pool()
+        self.assertNotEqual(StratisDbus.pool_list(), [])
+
+    def test_pool_create_same_name(self):
+        """
+        Test creating a pool that already exists.
+        """
+        pool_name = make_test_pool()
+        self.assertEqual(StratisDbus.pool_create(pool_name, DISKS)[1], 0)
+
+    def test_pool_destroy(self):
+        """
+        Test destroying a pool.
+        """
+        pool_name = make_test_pool()
+        self.assertEqual(StratisDbus.pool_destroy(pool_name)[1], 0)
+        self.assertEqual(StratisDbus.fs_list(), {})
+
+    def test_filesystem_create(self):
+        """
+        Test creating a filesystem.
+        """
+        pool_name = make_test_pool()
+        filesystem_name = fs_n()
+        self.assertEqual(
+            StratisDbus.filesystem_create(pool_name, filesystem_name)[1], 0
+        )
+
+    def test_filesystem_rename(self):
+        """
+        Test renaming a filesystem.
+        """
+        pool_name = make_test_pool()
+        filesystem_name = fs_n()
+        filesystem_name_rename = fs_n()
+        assert StratisDbus.filesystem_create(pool_name, filesystem_name)
+        self.assertEqual(
+            StratisDbus.filesystem_rename(
+                pool_name, filesystem_name, filesystem_name_rename
+            )[1],
+            0,
+        )
+
+    def test_filesystem_rename_same_name(self):
+        """
+        Test renaming a filesystem.
+        """
+        pool_name = make_test_pool()
+        filesystem_name = fs_n()
+        assert StratisDbus.filesystem_create(pool_name, filesystem_name)
+        self.assertEqual(
+            StratisDbus.filesystem_rename(pool_name, filesystem_name, filesystem_name)[
+                1
+            ],
+            0,
+        )
+
+    def test_filesystem_snapshot(self):
+        """
+        Test snapshotting a filesystem.
+        """
+        pool_name = make_test_pool()
+        filesystem_name = fs_n()
+        snapshot_name = fs_n()
+        assert StratisDbus.filesystem_create(pool_name, filesystem_name)
```

Comment:
Please make ```assert```s more specific. You want the assertion to fail if ```filesystem_create``` did not create the specified filesystem, but I think it will succeed regardless. I think this holds for all the ```assert```s in the tests that I can see.